versions/1/changelog.yml

```
@@ -10,6 +10,9 @@
   - description: Prepare for next version
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/265
+  - description: Support ausiliary files in terraform deploy spec
```

Nit. Should it be "auxiliary"?
```suggestion
  - description: Support auxiliary files in terraform deploy spec
```